cmd/jujud/agent/machine/manifolds.go

```diff
@@ -743,6 +748,8 @@ func Manifolds(config ManifoldsConfig) dependency.Manifolds {
 			PrometheusRegisterer: config.PrometheusRegisterer,
 			AgentName:            config.AgentName,
 			Clock:                config.Clock,
+			MuxShutdownWait:      config.MuxShutdownWait,
+			LogDir:               "/var/log/juju", //agentConfig.LogDir(),
```

Comment:
WIll this cause an issue for non-*nix builds?

Reply:
It probably isn't correct, but it is a valid directory on windows. It just happens to be "C:" or "D:" based on what drive the PWD is.

So its not great, and something like LogDir() or something based on DataDir() probably is better.